.eslintrc

```diff
@@ -1,9 +0,0 @@
-{
-  "extends": "plugin:bpmn-io/browser",
-  "rules": {
-    "no-restricted-imports": [ "error", {
-      "name": "inherits",
-      "message": "Use inherits-browser instead"
-    } ]
```

Comment:
Is this dropped?

Reply:
Part of core configuration, cf. https://github.com/bpmn-io/eslint-plugin-bpmn-io/blob/main/configs/browser.js#L15.